public/js/Game.js

```diff
@@ -0,0 +1,107 @@
+ function Game(){
+   this.scoreTotal = 0;
+   this.currentFrame = 1;
+   this.currentRoll = 1;
+   this.lastRollScore = 0;
+   this.pins = 10;
```

Comment:
Is it possible store every roll, and from that be able to derive `gameOver`, `currentRoll`, `currentFrame` etc from the rolls? This would reduce the coordination necessary inside Game.